xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -634,26 +639,38 @@ bool CDRMUtils::OpenDrm(bool needConnector)
 
       m_module = module;
 
-      CLog::Log(LOGDEBUG, "CDRMUtils::%s - opened device: %s using module: %s", __FUNCTION__, drmGetDeviceNameFromFd2(m_fd), module);
+      // store path and reset fd so the device can be reopened with logind
+      std::string path = drmGetDeviceNameFromFd2(m_fd);
+      close(m_fd);
+
+      m_fd = m_session->Open(path, 0);
+      if (m_fd < 0)
+        return false;
 
-      m_renderFd.attach(drmOpenWithType(module, nullptr, DRM_NODE_RENDER));
-      if (m_renderFd)
+      CLog::Log(LOGDEBUG, "CDRMUtils::%s - opened device: %s using module: %s", __FUNCTION__, path,
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)